unit-tests/tests.h

```diff
@@ -0,0 +1,104 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#ifndef TESTS_H
+#define TESTS_H
+
+#include "../include/librealsense2/rs.hpp"
+
+#define CATCH_CONFIG_MAIN
+
+#include "catch.h"
+
+#include <easylogging++.h>
+#ifdef BUILD_SHARED_LIBS
+// With static linkage, ELPP is initialized by librealsense, so doing it here will
+// create errors. When we're using the shared .so/.dll, the two are separate and we have
+// to initialize ours if we want to use the APIs!
+INITIALIZE_EASYLOGGINGPP
+#endif
+
+inline rs2::device_list find_device_by_product_line_or_exit( int product )
+{
+    rs2::context ctx;
+    rs2::device_list devices_list = ctx.query_devices( product );
+    size_t device_count = devices_list.size();
+    if( device_count == 0 )
+    {
+        std::cout << "No L500 device connected ";
+        exit( 0 );
+    }
+
+    return devices_list;
+}
+
+inline rs2::depth_sensor get_depth_sensor( rs2::device dev )
+{
+    auto sensors = dev.query_sensors();
+
+    auto it = std::find_if( sensors.begin(), sensors.end(), []( rs2::sensor s ) {
+        return s.is< rs2::depth_sensor >();
+    } );
+
+    return *it;
+}
+
+inline void remove_all_streams_arrived( rs2::frame f,
```

Comment:
Why is this here and not inside functional?

Reply:
Moved both  remove_all_streams_arrived() and find_devices_by_product_line_or_exit() to "functional.h" they both belong to live tests